app/queries/create_vimeo_video_mutator.rb

```diff
@@ -10,7 +10,7 @@ def create_vimeo_video
     response = vimeo_api.create_video(size, title, description)
 
     if response[:error].present? || response[:error_code].present?
-      if response[:developer_message].present?
+      if response[:developer_message].present?app/services/vimeo/api_service.rb
```

Comment:
@bodhish I just observed this file path here. Shouldn't this be cleaned up?

Reply:
Yes, this is a mistake

Reply:
Fixed.